### PR TITLE
Finance - session variable refactoring (WIP)

### PR DESCRIPTION
**Pull request by hmerrettICHK**
**Description**
First set of session variable changes for Finance module

**Motivation and Context**

**How Has This Been Tested?**

**Screenshots**

### CODE REVIEW

**Comment by hmerrettICHK** [comment]
Hi, Sandra! This PR is now complete - take a look when you can!

File: modules/Finance/expenses_manage_processBulk.php [hunk 1]
```diff
@@ -27,9 +27,9 @@
 
 if ($gibbonFinanceBudgetCycleID == '' or $action == '') { echo 'Fatal error loading this page!';
 } else {
-    $URL = $_SESSION[$guid]['absoluteURL'].'/index.php?q=/modules/'.getModuleName($_POST['address'])."/expenses_manage.php&gibbonFinanceBudgetCycleID=$gibbonFinanceBudgetCycleID";
+    $URL = $session->get('absoluteURL').'/index.php?q=/modules/'.getModuleName($_POST['address'])."/expenses_manage.php&gibbonFinanceBudgetCycleID=$gibbonFinanceBudgetCycleID";
 
-    if (isActionAccessible($guid, $connection2, '/modules/Finance/expenses_manage.php') == false) {
+    if (isActionAccessible($session, $connection2, '/modules/Finance/expenses_manage.php') == false) {
```
Comment (SKuipers) [inline modules/Finance/expenses_manage_processBulk.php:32]: Hi Harry, please check the syntax here.

**Review by SKuipers — CHANGES_REQUESTED** [review]
Looking good overall. I've spotted one area to check out. I haven't provided the solution because I'd like you to take a look and see if you can work out why it would generate an error. Be sure you're testing these, from what I can see this would cause a fatal error.

File: modules/Finance/expenses_manage_processBulk.php [hunk 2]
```diff
@@ -41,7 +41,7 @@
             $partialFail = false;
             //Export
             if ($action == 'export') {
-                $_SESSION[$guid]['financeExpenseExportIDs'] = $gibbonFinanceExpenseIDs;
+                $session->set('financeExpenseExportIDs') = $gibbonFinanceExpenseIDs;
```
Comment (SKuipers) [inline modules/Finance/expenses_manage_processBulk.php:44]: The first one looks good now, but the syntax here still needs some work

**Review by SKuipers — CHANGES_REQUESTED** [review]
Hi Harry, you caught one error in the syntax but the $session->set() syntax still needs some work. Keep at it!

**Review by SKuipers — APPROVED** [review]
Looks good 👍

**Pull request merged**